plugins/builds/steps/get.js

```diff
@@ -21,15 +21,15 @@ module.exports = () => ({
             }
         },
         handler: (request, reply) => {
-            const factory = request.server.app.buildFactory;
+            const buildFactory = request.server.app.buildFactory;
 
-            factory.get(request.params.id)
-                .then((model) => {
-                    if (!model) {
+            buildFactory.get(request.params.id)
+                .then((buildModel) => {
+                    if (!buildModel) {
                         throw boom.notFound('Build does not exist');
                     }
 
-                    const stepModel = model.steps.filter(step => (
+                    const stepModel = buildModel.steps.filter(step => (
```

Comment:
won't this only fetch the initial step data which `buildModel` has ?

Reply:
No, Min's model changes will call the step model and set the step information in a build when this get is called.